js/ui/windowManager.js

```diff
@@ -1776,14 +1776,20 @@ var WindowManager = new Lang.Class({
                                onCompleteParams: [shellwm, actor],
                                onOverwrite: onOverwrite,
                                onOverwriteScope: this,
-                               onOverwriteParams: [shellwm, actor]
+                               onOverwriteParams: [shellwm, actor],
+                               onUpdate: this._clipActor,
+                               onUpdateParams: [Clutter.ContentGravity.TOP, endY, actor],
```

Comment:
ClutterContentGravity has a very specific use case, and using it like this is very weird to me, but meh.

Reply:
@GeorgesStavracas If there's another enum that we can already use, then let us know. Otherwise, at least IMO, it's fine to use this and it's probably not worth creating yet another 4 positions enum.

Reply:
There is no better alternative in Clutter (and this isn't an actionable comment)